examples/reference/custom_components/AnyWidgetComponent.ipynb

```diff
@@ -306,9 +305,182 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2e7c1209-2b86-433a-8dd2-ca351ac02979",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import panel as pn\n",
+    "\n",
+    "from panel.custom import AnyWidgetComponent, Child\n",
+    "\n",
+    "\n",
+    "class Example(AnyWidgetComponent):\n",
+    "\n",
+    "    child = Child(class_=pn.pane.Markdown)\n",
+    "\n",
+    "    _esm = \"\"\"\n",
+    "    function render({ model, el }) {\n",
+    "      const button = document.createElement(\"button\");\n",
+    "      button.append(model.get_child(\"child\"))\n",
+    "      el.appendChild(button);\n",
+    "    }\n",
+    "\n",
+    "    export default { render };\n",
+    "    \"\"\"\n",
+    "\n",
+    "\n",
+    "Example(child=pn.panel(\"A **Markdown** pane!\")).servable()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e7da88d-aaa6-41ba-8a33-a0f93e0eb302",
+   "metadata": {},
+   "source": [
+    "The `class_` argument also supports a tuple of types:\n",
+    "\n",
+    "```python\n",
+    "    child = Child(class_=(pn.pane.Markdown, pn.pane.HTML))\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "53ce78f8-b4f5-4ff6-afe0-563efa0d3fc7",
+   "metadata": {},
+   "source": [
+    "## Displaying a List of Children\n",
+    "\n",
+    "You can also display a `List` of `Viewable` objects using the `Children` parameter type:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "634d84e4-e372-4cff-b40e-7e16df179765",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import panel as pn\n",
+    "\n",
+    "from panel.custom import AnyWidgetComponent, Children\n",
+    "\n",
+    "pn.extension()\n",
+    "\n",
+    "\n",
+    "class Example(AnyWidgetComponent):\n",
+    "\n",
+    "    objects = Children()\n",
+    "\n",
+    "    _esm = \"\"\"\n",
+    "    function render({ model, el }) {\n",
+    "      const div = document.createElement('div')\n",
+    "      div.append(...model.get_child(\"objects\"))\n",
+    "      el.appendChild(div);\n",
+    "    }\n",
+    "\n",
+    "    export default { render };\n",
+    "    \"\"\"\n",
+    "\n",
+    "\n",
+    "Example(\n",
+    "    objects=[pn.panel(\"A **Markdown** pane!\"), pn.widgets.Button(name=\"Click me!\"), {\"text\": \"I'm shown as a JSON Pane\"}]\n",
+    ").servable()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "748776ca-925d-44dd-b503-3823252165c9",
+   "metadata": {},
+   "source": [
+    ":::note\n",
+    "You can change the `item_type` to a specific subtype of `Viewable` or a tuple of\n",
+    "`Viewable` subtypes.\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91a8cb50-c38d-4d8a-b992-7bf013926a73",
+   "metadata": {},
+   "source": [
     "### React\n",
     "\n",
     "You can use React with `AnyWidget` as shown below."
```

Comment:
The `React` example works on the server but not in the notebook

![image](https://github.com/user-attachments/assets/182c097e-3fe2-4d49-a077-b668f28bb5e1)

```python
import panel as pn
import param

from panel.custom import AnyWidgetComponent

pn.extension()


class CounterButton(AnyWidgetComponent):

    value = param.Integer()

    _importmap = {
        "imports": {
            "@anywidget/react": "https://esm.sh/@anywidget/react",
            "react": "https://esm.sh/react",
        }
    }

    _esm = """
    import * as React from "react"; /* mandatory import */
    import { createRender, useModelState } from "@anywidget/react";

    const render = createRender(() => {
      const [value, setValue] = useModelState("value");
      return (
        <button onClick={() => setValue(value + 1)}>
          count is {value}
        </button>
      );
    });
    export default { render }
    """

CounterButton().servable()
```